react/src/components/Table/Table.tsx

```diff
@@ -71,34 +72,103 @@ const resolveSex = (sexPhenotype: string) => {
     } else return 'Unknown';
 };
 
-/* flatten data and compute values as needed (note that column display formatting function should not alter values for ease of export) */
-const prepareData = (queryResult: VariantQueryDataResult[]): ResultTableColumns[] => {
-    return queryResult.flatMap(d => {
-        if (d.variant.callsets.length) {
-            //one row per individual per callset
-            return d.variant.callsets
-                .filter(cs => cs.individualId === d.individual.individualId)
-                .map(cs =>
-                    addAdditionalFieldsAndFormatNulls({
-                        ...cs.info,
-                        ...flattenBaseResults(d),
-                    })
-                );
+/* 1, Sort queryResult in ascending order according to variant's chromosome, ref, alt, start,end. 
+2, flatten data and compute values as needed 
+(note that column display formatting function should not alter values for ease of export) */
+const prepareData = (queryResult: VariantQueryDataResult[]): [ResultTableColumns[], number[]] => {
+    const sortedQueryResult = [...queryResult].sort(function (a, b) {
+        if (a.variant.referenceName < b.variant.referenceName) {
```

Comment:
Can we move this `sort` functionality to `tableHelpers.ts`? Ideally, we want Table.tsx to focus on styling code and React logic and abstract the logic away to another file.